authors/apps/comments/models.py

```diff
@@ -29,3 +29,12 @@ def __str__(self):
         return self.reply_body
 
 
+class CommentLike(models.Model):
+    comment=models.ForeignKey(Comment,on_delete=models.CASCADE)
+    like_status=models.BooleanField()
+    liked_by=models.ForeignKey(Profile,on_delete=models.CASCADE)
+
+    def __str__(self):
+        return "like by " + " " + str(self.liked_by)
```

Comment:
@MuhweziDeo you can use the format method here instead of concatenating the strings.